src/client/components/pages/entity-revisions.js

```diff
@@ -16,6 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+
```

Comment:
It was just one line addition so i thought it to include it in this PR? Should I remove it and send a separate PR?